crates/p2p_proto/proto/common.proto

```diff
@@ -146,3 +169,7 @@ message BlockBody {
   repeated Transaction transactions = 1;
   repeated Receipt receipts = 4;
 }
+
+message CompressedContractClass {
+  bytes class = 1;
```

Comment:
This was the simples way to move forward, I'm not that convinced we need to litter the file with all the peculiarities of cairo 1&0 contract defs.

Reply:
Yeah I think its good. Especially just to get going. We can always add things if we find we need them.

Reply:
No reason really, I went with whatever I could get from the db out of the box and not define any more proto messages

Reply:
From what I recall, the compression should be a transport / network level issue? So just having the uncompressed definition is probably fine. Otherwise we have to agree on compression format etc.

Reply:
Lemme keep this as is for the time being and rework it in the next PR when storage<>sync handlers are added. :thinking: